prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -713,4 +732,157 @@ class ActionHistory {
             }
         }
     }
+
+    fun createResponseBody(
```

Comment:
You need Kdoc for all your classes, functions and attributes.

Reply:
Can do - this was just moving this function from ReportFunction.kt to ActionHistory (did not have kdoc in ReportFunction).  I'll review again and add missing kdocs.

Reply:
Updated